pkg/bff/models/v1/registration.go

```diff
@@ -118,9 +119,59 @@ func (r *RegistrationForm) Validate(step StepType) error {
 }
 
 // Validate only the fields in the basic details step.
-func (r *RegistrationForm) ValidateBasicDetails() ValidationErrors {
-	// TODO: update basic details validation
-	return nil
+func (r *RegistrationForm) ValidateBasicDetails() (v ValidationErrors) {
+	// Validate website
+	// TODO: Check if this is a valid URL?
+	if strings.TrimSpace(r.Website) == "" {
```

Comment:
I wonder if we should do a preprocessing step for things like trimming whitespace - or maybe we should do that in Update?